xdrip/Storyboards/de.lproj/AlertTypesSettingsView.strings

```diff
@@ -1,8 +1,8 @@
-/// when editing the snoozeperiod of an alert type, 
-"alerttypesettingsview_givesnoozeperiod" = "Schlummern für Minuten";
+/// when editing the snoozeperiod of an alert type, displayed is the time until snooze should be active
+"alerttypesettingsview_givesnoozeperiod" = "Schlummern bis";
```

Comment:
This is not correct actually. The snoozeperiod is how long it should be snoozed in minutes (and not until when it should be snoozed)

Reply:
I guess here we have a misunderstanding. Snoozing period is in Minutes.

But when you set the snozzing to 10 minutes, you get the time back until when the alarm is snoozed:

So in this case the translation can't be snoozing for minutes when the result is a time in the future instead of minutes.

![IMG_2805](https://user-images.githubusercontent.com/78857191/112206707-87c5fa00-8c16-11eb-9c17-c4d256a0ad53.PNG)
![IMG_2806](https://user-images.githubusercontent.com/78857191/112206724-8d234480-8c16-11eb-8fac-58c616e9c680.PNG)

Reply:
<just double check the english version, there it's exactly the same.... snoozing time is choosen in minutes, but after choosing 10 minutes the result is the time from now + choosen snoozing time.